packages/typo3-docs-theme/resources/template/body/menu/mainmenu/menu-level.html.twig

```diff
@@ -1,19 +1,16 @@
-{%- apply spaceless -%}
-        <li class="toctree-l{{ menuEntry.level }} {%- if menuEntry.isInRootline %} current {% endif %}">
-            <a href="{{ renderLink(menuEntry.url) }}"
-               class="nav-link {% if menuEntry.options.active %} active {% endif %}"
-                {%- if menuEntry.options.active %}  aria-current="page" {% endif -%} >
-                {{ renderNode(menuEntry.value.value) }}
-            </a>
+<li class="toctree-l{{ menuEntry.level }} {%- if menu.currentPath == entry.url %} current {%- endif -%}{% if entry.url in menu.rootlinePaths %} active {%- endif -%}">
```

Comment:
Why use class "current" when no style is attached to it?

Reply:
I think that the menu needs a new and thoughtful HTML structure anyway and thought we can do that in a follow up.

Reply:
I don't like loose ends. But when you come with a follow-up in the near time, it's fine.